src/Tests/Utils/ConfigReaderTest.php

```diff
@@ -37,14 +37,14 @@ class ConfigReaderTest extends \PHPUnit_Framework_TestCase {
 	/**
 	 * @return array
 	 */
-	public function testGetByPathProvider(){
+	public function getByPathProvider(){
```

Comment:
`phpunit` was running this a a test, because the method name started with `test`
It gave:
```
There was 1 risky test:

1) Owncloud\Updater\Tests\Utils\ConfigReaderTest::testGetByPathProvider
This test did not perform any assertions
```